tests/menu_items_registry.test.ts

```diff
@@ -146,6 +147,37 @@ describe("Menu Item actions", () => {
     expect(getCellContent(model, "A1")).toEqual("");
   });
 
+  test("'Edit -> paste' if copied from grid and content altered before paste", async () => {
+    setCellContent(model, "A1", "a1");
+    doAction(["edit", "copy"], env); // first copy from grid
+    setCellContent(model, "A1", "os clipboard");
+    selectCell(model, "C3");
+    doAction(["edit", "paste"], env);
+    await nextTick();
```

Comment:
nextTick() useless I think

Reply:
It seems (referring to other test cases) after every single doAction performed, we have to add an `await nextTick()` before expecting a command dispatch for that action. I tried removing them as you suggested, but all 4 test cases failed...

Reply:
Yeah the problem seems that the function `paste` called by the action is async, so we indeed need the `await nextTick();`. Maybe we should change the design and alway sawait the actions, but don't worry about it then.

Reply:
Fixed it in https://github.com/odoo/o-spreadsheet/pull/2522, you should remove the nextTick and await the `doAction` when it's merged and you rebase your branch :)

Reply:
Cool :)